### DISCUSSION TITLE
Bad package: zip-local 0.3.5

### DISCUSSION THREAD

**Discussion by qphoria** [npm/feedback#739] [Registry] [answered]
Not sure if this is how you report issues with the registry repo but noticing that zip-local 0.3.5 is a bad package. 0.3.4 works but not 0.3.5

You can download the tarball manually to see
https://registry.npmjs.org/zip-local/-/zip-local-0.3.5.tgz

**Comment by MylesBorins** [comment, marked as answer]
Hey @qphoria. Unless there is a problem specifically with the deliver of the package itself, such as a mismatch in integrity or an inability to download the package, there isn't anything that we as npm can do.

I used `npm pack zip-local` to grab the latest version and verify the integrity of the package as served by the registry and everything appears normal. If the package itself is broken due to programming error you will have to take that up with the package maintainer [on their repostiory](https://github.com/Mostafa-Samir/zip-local)
